farmdata2_modules/fd2_tabs/fd2_barn_kit/transplantingReport.html

```diff
@@ -0,0 +1,639 @@
+                    //this.reportVisible = false
+                    //this.transplantingLogs = []
+                },
+                endDateChange(selectedDate){
+                    this.endDate = selectedDate
+                    //this.reportVisible = false
+                    //this.transplantingLogs = []
+                },
+                seedingLogRequestWithDates(){
+                    this.reportVisible = true
+                    let link = '/log.json?type=farm_transplanting&timestamp[ge]=' + dayjs(this.startDate).unix() + '&timestamp[le]=' + dayjs(this.endDate).unix()
+
+                    getAllPages(link, this.transplantingLogs)
+                        .then(() => {
+                            this.stillLoading = false
+                        })
+                        .catch((err) => { 
+                            this.errBanner = true
+                            var errorBanner = document.getElementById('scrollBanners')
+                            var pageHeader = document.getElementById('navbar')
+                            var headerBounds = pageHeader.getBoundingClientRect()
+                            var alertBounds = errorBanner.getBoundingClientRect()
+                            scrollBy({
+                                top: alertBounds.top - headerBounds.bottom,
+                                behavior: 'smooth'
+                            })
+                        })
+                    this.stillLoading = true
+                },
+                cropChange(selectedCrop){
+                    this.selectedCrop = selectedCrop
+                },
+                areaChange(selectedArea){
+                    this.selectedArea = selectedArea
+                },
+                configChange(){
+                    if(this.configToIDMap.labor == 'Hidden') {
+                        this.visibleColumns[7] = false
+                        this.visibleColumns[8] = false
+                    }
+                },
+                /**
+                 * updates the log that the user saved
+                 */
+                updateRow(updateObject, id){
+                    this.updateLog = this.transplantingLogs.filter((x) => x.id === id)
+                    let updateSeedingObject = {}
+                    let updatePlantObject = {}
+                    
+                    /**
+                     * The Table component returns an object with the cells that were updated. We don't know how many or wich ones where updated, so we check for all possible cells that could have been updated.
+                     */
+                    let keys = Object.keys(updateObject)
+                    if(keys.includes('Hours') || keys.includes('Workers') || keys.includes('Bed Feet') || keys.includes('Row Feet') ||
+                        keys.includes('Rows/Bed') ||   keys.includes('Trays')){
+                            updateSeedingObject['quantity'] = this.quantityTransplantSeeding
```

Comment:
Shouldn't this be updating the transplanting not the seeding?  Maybe this is just semantics and a copy and paste issue from the seeding report.  But names should be updated throughout to be consistent with the purpose of the page, otherwise it is quite confusing for another reader.